packages/manager/cypress/e2e/firewalls/create-firewall.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { createLinode } from '@linode/api-v4/lib/linodes';
+import { createLinodeRequestFactory } from 'src/factories/linodes';
+import { regions } from 'support/constants/regions';
+import { authenticate } from 'support/api/authentication';
+import { containsClick, getClick } from 'support/helpers';
+import { interceptCreateFirewall } from 'support/intercepts/firewalls';
+import { randomItem, randomString, randomLabel } from 'support/util/random';
+
+authenticate();
+describe('create firewall', () => {
+  /*
+   * - Creates a firewall that is not assigned to a Linode.
+   * - Confirms that an error message appears upon submitting without a label.
+   * - Confirms that firewall is listed correctly on firewalls landing page.
+   * - Confirms that tags exist on firewall.
+   */
+  it('creates a firewall without a linode', () => {
+    const firewall = {
+      label: randomLabel(),
+    };
+
+    interceptCreateFirewall().as('createFirewall');
+    cy.visitWithLogin('/firewalls/create');
+
+    // An error message appears when attempting to create a Firewall without a label
+    getClick('[data-testid="create-firewall-submit"]');
+    cy.findByText('Label is required.');
+
+    // Fill out and submit firewall create form.
+    containsClick('Label').type(firewall.label);
+    getClick('[data-testid="create-firewall-submit"]');
```

Comment:
There's some potential for flake here if this test runs on an account which already has one or more Firewalls -- `containsClick('Label')` will select both the _Label_ table column header on the landing page, and the _Label_ text field in the _Create Firewall_ drawer.

We can fix this by selecting the _Create Firewall_ drawer and only making these assertions in the context of that drawer:

```suggestion
    ui.drawer
      .findByTitle('Create Firewall')
      .should('be.visible')
      .within(() => {
        // An error message appears when attempting to create a Firewall without a label
        getClick('[data-testid="create-firewall-submit"]');
        cy.findByText('Label is required.');

        // Fill out and submit firewall create form.
        containsClick('Label').type(firewall.label);
        getClick('[data-testid="create-firewall-submit"]');
      });
```

(This also applies to lines `72`-`76`, and you'll need to import the `ui` object from `support/ui` to use the drawer helper)